webapp/src/main/webapp/components/snackbar/ExoNewsNotificationAlerts.vue

```diff
@@ -47,6 +47,16 @@ export default {
         });
       }
     });
+    this.$root.$on('news-shared', (news, spaces) => {
+      if (news && news.newsId && spaces && spaces.length > 0) {
+        let message = this.$t('news.share.message');
+        spaces.forEach(space => message += `${space}, `);
```

Comment:
```suggestion
        spaces.forEach(space => message = `${message } ${space} `);
```
I think that we should not display "," because you will have a usless "," at the end

Reply:
How about `message = spaces.join(', ');` ?

Reply:
yep , thanks a lot houssem